lib/read.js

```diff
@@ -10,9 +11,10 @@ const schema  = require('./schema');
  * @param {String|ReadableStream} file File where the torrent
  *   resides, can be local file, remote, or a readable stream.
  * @param {Function(!Error, Torrent)} callback
+ * @param {Boolean} validate validate or not schema
  * @param {ReadableStream}
  */
-module.exports = (file, callback) => {
+module.exports = (file, callback, validate = true) => {
```

Comment:
typically, the `callback` is the last argument in a function. you can make `validate` an optional param by doing something like

```js
if (typeof validate === 'function') {
  callback = validate
  validate = true
}

Reply:
has been done